.circleci/config.yml

```diff
@@ -34,34 +34,74 @@ jobs:
           command: |
             python --version
             pip --version
-            pip install --user -r requirements-dev.txt
+            pip install --progress-bar off --user -r requirements-dev.txt
+            pip install --progress-bar off codecov
             python setup.py build
 
       - run:
           name: Run tests
           command: |
             python -m coverage run -m unittest discover tests/ -v
-            python -m coverage report
+            python -m coverage report -i
+            codecov
 
       - store_artifacts:
           path: test-reports
           destination: test-reports
 
-  test-3.7:
+  test-py37:
     <<: *test-template
     docker:
       - image: circleci/python:3.7
 
-  test-3.6:
+  test-py36:
     <<: *test-template
     docker:
       - image: circleci/python:3.6
 
-  test-3.5:
+  test-py35:
     <<: *test-template
     docker:
       - image: circleci/python:3.5
 
+  test-integration:
+    docker:
+      - image: continuumio/miniconda3
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install compilers
+          command: |
+            source activate base
+            conda install -c conda-forge gcc_linux-64 gxx_linux-64 --yes --quiet
```

Comment:
What do we need these compilers for? fresnel and hoomd?

Reply:
They're for compiling the DCD reader, which is Cythonized into C++. By default this image has no compilers but the other image we use has compilers. 😋

Reply:
Ah, that makes sense.